styles/index.css

```diff
@@ -0,0 +1,56 @@
+img {
+  width: 18%;
+  height: auto;
+  vertical-align: middle;
+}
+
+main {
+  margin-bottom: 5%;
+}
+
+.link-grid {
+  display: grid;
+  grid-template-columns: 5% 20% 20% 6% 20% 20% 9%;
+  grid-template-rows: 20% 20% 20% 20% 20%;
+}
+
+.gallery-text {
+  grid-column: 2/4;
+}
+
+.fun-facts-text {
+  grid-column: 5/7;
+}
+
+.link-gallery {
+  grid-row: 2/4;
+  grid-column: 2/4;
+}
+
+.link-facts {
+  grid-row-start: 2/4;
+  grid-column: 5/7;
+}
+
+.link-facts>img {
+  object-fit: cover;
+  width: 100%;
+  border: 10px solid rgba(138, 101, 199, 0.352);
+  border-radius: 5px;
+  padding: 0px;
+  margin: 20px;
+}
+
+.link-gallery>img {
```

Comment:
For readability, make sure to put spaces between your selectors! 

       ".link-gallery > img"